Recipes Project.py

```diff
@@ -1,2 +1,23 @@
-x = 2+2
-print (x)
+#asking for available ingredients"""
+ingredients = input("what ingredients do you have available? (please seperate each with a comma)")
+print ('okay let me check what recipes we have available with', ingredients, '...')
+
+
+#importing recipes and store in dict varibale
+import csv
```

Comment:
for style purposes, normally, people like to import all packages at the start of the script. But it is just a styling suggestion and style is subjective (maybe). :P